src/boardlib/api/aurora.py

```diff
@@ -391,3 +361,224 @@ def save_climb(
+                'board': ascent_row['board'],
+                'angle': ascent_row['angle'],
+                'climb_name': ascent_row['name'],
+                'date': ascent_row['date'],
+                'logged_grade': ascent_row['logged_grade'],
+                'displayed_grade': ascent_row.get('displayed_grade', None),
+                'difficulty': ascent_row['difficulty'],
+                'tries': total_tries,
+                'is_mirror': ascent_row['is_mirror'],
+                'is_ascent': True,
+                'comment': ascent_row['comment']
+            })
+            bids_summary = bids_summary.drop(bid_match.index)
+        else:
+            final_logbook.append({
+                'board': ascent_row['board'],
+                'angle': ascent_row['angle'],
+                'climb_name': ascent_row['name'],
+                'date': ascent_row['date'],
+                'logged_grade': ascent_row['logged_grade'],
+                'displayed_grade': ascent_row.get('displayed_grade', None),
+                'difficulty': ascent_row['difficulty'],
+                'tries': ascent_row['tries'],
+                'is_mirror': ascent_row['is_mirror'],
+                'is_ascent': True,
+                'comment': ascent_row['comment']
+            })
+
+    for _, bid_row in bids_summary.iterrows():
+        if db_path:
+            difficulty = get_difficulty_from_db(db_path, bid_row["climb_uuid"], bid_row["angle"])
+            displayed_grade = convert_difficulty_to_grade(difficulty, grades_dict, grade_type)
+        else:
+            displayed_grade = None
+            difficulty = None
+        
+        final_logbook.append({
+            'board': bid_row['board'],
+            'angle': bid_row['angle'],
+            'climb_name': bid_row['climb_name'],
+            'date': bid_row['date'],
+            'logged_grade': None,
+            'displayed_grade': displayed_grade,
+            'difficulty': difficulty,
+            'tries': bid_row['tries'],
+            'is_mirror': bid_row['is_mirror'],
+            'is_ascent': False,
+            'comment': bid_row.get('comment', None)  # Use .get() to safely handle missing 'comment'
+        })
+    return final_logbook
+
+
+def calculate_sessions_count(group):
+    group = group.sort_values(by='date')
+    unique_dates = group['date'].dt.date.drop_duplicates().reset_index(drop=True)
+    sessions_count = unique_dates.rank(method='dense').astype(int)
+    sessions_count_map = dict(zip(unique_dates, sessions_count))
+    group['sessions_count'] = group['date'].dt.date.map(sessions_count_map)
+    return group
+
+
+def calculate_tries_total(group):
+    group = group.sort_values(by='date')
+    group['tries_total'] = group['tries'].cumsum()
+    return group
+
+
+def logbook_entries(board, username, password, grade_type="font", db_path=None):
+    login_info = login(board, username, password)
+    token = login_info["token"]
+    user_id = login_info["user_id"]
```

Comment:
can we go straight for the token here instead of username, password? For climbdex we only save the bearer token after the initial login